torchaudio/models/wav2vec2/pretrained.py

```diff
@@ -160,6 +161,82 @@ def _get_labels():
 [`Source <https://github.com/pytorch/fairseq/tree/main/examples/hubert#pre-trained-and-fine-tuned-asr-models>`__]
 """
 
+HUBERT_LARGE = Wav2Vec2PretrainedModelBundle(
+    'hubert_fairseq_large_ll60k.pth',
+    {
+        'extractor_mode': 'layer_norm',
+        'extractor_conv_layer_config': [
+            (512, 10, 5),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 2, 2),
+            (512, 2, 2),
+        ],
+        'extractor_conv_bias': False,
+        'encoder_embed_dim': 1024,
+        'encoder_projection_dropout': 0.0,
+        'encoder_pos_conv_kernel': 128,
+        'encoder_pos_conv_groups': 16,
+        'encoder_num_layers': 24,
+        'encoder_num_heads': 16,
+        'encoder_attention_dropout': 0.0,
+        'encoder_ff_interm_features': 4096,
+        'encoder_ff_interm_dropout': 0.0,
+        'encoder_dropout': 0.0,
+        'encoder_layer_norm_first': True,
+        'encoder_layer_drop': 0.0,
+        'aux_num_out': None,
+    },
+    _labels=None,
+)
+HUBERT_LARGE.__doc__ = """HuBERT model with "Large" configuration.
+
+Trained on 60,000 hours of *Libri-Light* [:footcite:`librilight`] dataset.
+
+Originally published by the authors of *HuBERT* [:footcite:`hsu2021hubert`].
+[`Source <https://github.com/pytorch/fairseq/tree/main/examples/hubert#pre-trained-and-fine-tuned-asr-models>`__]  # noqa: E501
```

Comment:
nit: this comment unintentionally shows up in the docs as well
https://359154-90321822-gh.circle-artifacts.com/0/docs/models.html#torchaudio.models.Wav2Vec2PretrainedModelBundle.HUBERT_LARGE